vue3-ts-quasar/src/composables/github/useGists.ts

```diff
@@ -0,0 +1,21 @@
+import { Ref } from 'vue';
+import { parseQuery } from '@/components/GistsPanel/parseQuery';
+import { useQuery, useResult } from '@vue/apollo-composable';
+import { USER_GISTS_QUERY } from './queries';
+import { GistItem } from '../github/types';
+
+interface UseGists {
```

Comment:
We can move all individual related types to `src/composables/github/types/Gist.ts` and export them as interfaces/types from the same. 

Not sure if we should move the query type to `src/composables/github/types/Gist.ts`  or into `src/composables/github/queries/user-gists.query.ts` . What do you think @WillHutt ?